File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/settings-new/shared/DTOs/price-category.dto.ts
export class PriceCategory_DTO {

    public PriceCategoryId: number = 0;
    public OldPriceCategoryId: number = 0;
    public NewPriceCategoryId: number = 0;
    public PriceCategoryCode: string = '';
    public PriceCategoryName: string = '';
    public Description: string = '';
    public ShowInRegistration: boolean = false;
    public ShowInAdmission: boolean = false;
    public IsDefault: boolean = false;
    public IsRateDifferent: boolean = false;
    public CreatedBy: number = 0;
    public CreatedOn: string = '';
    public IsActive: boolean = true;
    public PharmacyDefaultCreditOrganizationId: number = 0;
    public IsPharmacyRateDifferent: boolean = false;
    public DisplaySequence: number = 0;
    public IsCoPayment: boolean = false;
    public DefaultCreditOrganizationId: number = 0;
    public Copayment_CashPercent: number = 0;
    public Copayment_CreditPercent: number = 0;
}

File: Code/Websites/DanpheEMR/Controllers/Settings/DTO/PriceCategoryDTO.cs
namespace DanpheEMR.Controllers.Settings.DTO
{
    public class PriceCategoryDTO
    {
        public int PriceCategoryId { get; set; }
        public string PriceCategoryCode { get; set; }
        public string PriceCategoryName { get; set; }
        public string Description { get; set; }
        public bool IsDefault { get; set; }
        public bool IsActive { get; set; }
        public bool ShowInRegistration { get; set; }
        public bool ShowInAdmission { get; set; }
    }
}
